src/crimson/tools/store-nbd.cc

```diff
@@ -345,31 +324,27 @@ int main(int argc, char** argv)
 }
 
 class nbd_oldstyle_negotiation_t {
-  uint64_t magic = ntohll(0x4e42444d41474943); // "NBDMAGIC"
-  uint64_t magic2 = ntohll(0x00420281861253);  // "IHAVEOPT"
+  uint64_t magic2 = seastar::cpu_to_be(0x00420281861253);  // "IHAVEOPT"
```

Comment:
@athanatos Sam, i think the packet definition for communication between nbd client and server should be in network endian (be). so i define this the other way around.